src/Fingerprint.ServerSdk.Examples/Program.cs

```diff
@@ -0,0 +1,113 @@
+﻿using System.Text.Json;
+using Fingerprint.ServerSdk.Api;
+using Fingerprint.ServerSdk.Client;
+using Fingerprint.ServerSdk.Extensions;
+using Fingerprint.ServerSdk.Model;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace Fingerprint.ServerSdk.Examples;
+
+public static class Program
+{
+    public static async Task Main(string[] args)
+    {
+        var host = CreateHostBuilder(args).Build();
+        var api = host.Services.GetRequiredService<IFingerprintApi>();
+
+        await SearchEventsExample(api);
+
+        var requestId = Environment.GetEnvironmentVariable("REQUEST_ID");
+        if (!string.IsNullOrWhiteSpace(requestId))
+        {
+            await GetEventExample(api, requestId);
+        }
+
+        var updateRequestId = Environment.GetEnvironmentVariable("UPDATE_REQUEST_ID");
+        if (!string.IsNullOrWhiteSpace(updateRequestId))
+        {
+            await UpdateEventExample(api, updateRequestId);
+        }
+
+        var visitorId = Environment.GetEnvironmentVariable("VISITOR_ID");
+        if (!string.IsNullOrWhiteSpace(visitorId))
+        {
+            await DeleteVisitorDataExample(api, visitorId);
+        }
+    }
+
+    private static IHostBuilder CreateHostBuilder(string[] args) => Host.CreateDefaultBuilder(args)
+        .ConfigureApi((_, _, options) =>
+        {
+            var token = Environment.GetEnvironmentVariable("SECRET_API_KEY");
+            if (string.IsNullOrWhiteSpace(token))
+            {
+                throw new InvalidOperationException("SECRET_API_KEY is missing");
+            }
+
+            var region = Environment.GetEnvironmentVariable("REGION");
+            if (!string.IsNullOrWhiteSpace(region))
+            {
+                options.Region = Regions.Parse(region);
+            }
+
+            var bearerToken = new BearerToken(token);
+            options.AddTokens(bearerToken);
+        });
+
+    private static async Task SearchEventsExample(IFingerprintApi api)
+    {
+        var events = await api.SearchEventsAsync(2, bot: "bad");
```

Comment:
Let's use all params as named
```suggestion
        var events = await api.SearchEventsAsync(limit: 2, bot: "bad");
```

Reply:
I've migrated the other usages to named arguments as well. If we don't prefer this for parameters that already have the same names, let me know, and I can revert it. Updated with commit [#3ef3729](https://github.com/fingerprintjs/fingerprint-pro-server-api-dotnet-sdk/pull/126/commits/3ef3729860ca32afdbf9d0c044751da2a7dcbafc)